terraform/050-pw-manager/vars.tf

```diff
@@ -210,6 +210,12 @@ variable "mysql_user" {
   type = string
 }
 
+variable "password_rule_alpha_and_numeric" {
+  description = "require alpha and numeric characters in password, use \"false\" or \"true\" strings"
+  type        = string
+  default     = "false"
```

Comment:
Why not 'bool'?

Reply:
Because it's used in the ECS task definition, which only accepts `string`. Since you mention it, though, it could be a bool here and converted to string in the code. If I decide it's worth doing, I'll do it separately because there are probably many places that could be changed.